### DISCUSSION TITLE
Add "Bookmark" button to side-menu in small-screen mode.

### DISCUSSION THREAD

**Discussion by cromega08** [dailydotdev/daily#782] [Feature Requests]
Hello!

I've been using the daily.dev chrome extension for a while now and there was a time when the aforementioned button would show up in the menu when the window was not in fullscreen mode.

It was more accessible and easier to navigate.

- Images of request:

![small-screen mode](https://user-images.githubusercontent.com/90356347/206821144-87c3a64f-530a-4d61-bd00-bf6349530c8a.png)

![image](https://user-images.githubusercontent.com/90356347/206821920-553f6df9-26f0-4a2a-8f83-f309965fc1e8.png)

- Additional Information:

    - Browser: Brave - [Versión 1.46.140 Chromium: 108.0.5359.99 (Build oficial) (64 bits)](https://brave.com/latest/)

**Comment by idoshamun** [comment]
Hi @cromega08,

Thanks for the report. We will fix it.
